benchmarks/shelley3pools/tx-generator-script.json

```diff
@@ -2,27 +2,33 @@
     { "setInitCooldown": 10 },
     { "setNumberOfInputsPerTx": 1 },
     { "setNumberOfOutputsPerTx": 1 },
-    { "setNumberOfTxs": 500 },
+    { "setNumberOfTxs": 5000 },
     { "setTPSRate": 10 },
     { "setTxAdditionalSize": 0 },
     { "setFee": 0 },
     { "setTTL": 1000000 },
     { "startProtocol": "configuration/configuration-generator.yaml" },
-    { "setEra": "MaryEra" },
+    { "setEra": "AllegraEra" },
     { "setLocalSocket": "logs/sockets/1" },
     { "readSigningKey": "pass-partout", "filePath": "configuration/genesis-shelley/utxo-keys/utxo1.skey" },
     { "secureGenesisFund": "genFund", "genesisKey": "pass-partout", "fundKey": "pass-partout" },
     { "delay": null },
```

Comment:
What does `delay: null` mean?

Reply:
Just empty/no data. Needed for correct JSON syntax.

Reply:
Sorry I was unclear -- I mean what does the whole command mean?